components/push/android/src/main/java/mozilla/appservices/push/PushManager.kt

```diff
@@ -332,11 +337,13 @@ interface PushAPI : java.lang.AutoCloseable {
      *
      * @param channelID Channel ID (UUID4) for new subscription, either pre-generated or "" and one will be created.
      * @param scope Site scope string (defaults to "" for no site scope string).
+     * @param appServerKey VAPID public key to "lock" subscriptions (defaults to "" for no key)
      * @return a SubscriptionInfo structure
      */
     fun subscribe(
         channelID: String = "",
-        scope: String = ""
+        scope: String = "",
+        appServerKey: String = ""
```

Comment:
Since `appServerKey` is optional, we might also want to make this nullable `String?`.